docs/spec/attributes.decoder.md

```diff
@@ -5,7 +5,7 @@
 
 ~~~~~
 DecodeAttributesDecoderData(buffer) {
-  num_attributes                                                        I32
+  num_attributes                                                        varUI32
```

Comment:
do we have "varUI32" documented anywhere? (for example that it is decoded with  need to decode it with DecodeVarint<UI32>() function)